pkg/grpc/management/proto/management.proto

```diff
@@ -1356,7 +1356,7 @@ message Email {
 }
 
 message UniqueUserRequest {
-  string user_name = 1 [(validate.rules).string = {min_len: 1, max_len: 200}];
+  string user_name = 1 [(validate.rules).string.pattern = "^[[:graph:]]{1,200}$"];
```

Comment:
this regex doesn't allow `é` or `ä`

shouldn't we just check for spaces at the beginning / end and trim them or return an error
IMHO if you insert a space, emoji or any other (byte) value which doesn't break our system, i'm fine with it

Reply:
changed regex. now all characters should be allowd (adföäüèê😀sdf) is a valid name